src/hotspot/share/gc/epsilon/epsilon_globals.hpp

```diff
@@ -64,7 +64,7 @@
           "smaller TLABs until policy catches up.")                         \
                                                                             \
   product(bool, EpsilonElasticTLABDecay, true, EXPERIMENTAL,                \
-          "Use timed decays to shrik TLAB sizes. This conserves memory "    \
+          "Use timed decays to shrink TLAB sizes. This conserves memory "   \
```

Comment:
Let's not do the typo fixes in this PR. Maybe there are other spelling problems elsewhere in `gc/epsilon` that we could fix wholesale in another PR?

Reply:
OK, I will revert the typo fixes change.